invesalius/navigation/robot.py

```diff
@@ -156,11 +172,23 @@ def InitializeRobot(self):
         )
         print("Robot initialized")
 
+    def GetCoilName(self):
+        return self.coil_name
+
+    def SetCoilName(self, name):
+        self.coil_name = name
+        self.SaveConfig("robot_coil", name)
+
     def SendTargetToRobot(self):
         # If the target is not set, return early.
         if self.target is None:
             return False
 
+        navigation = self.navigation
+        # XXX: These are needed for computing the target in tracker coordinate system. Ensure that they are set.
+        if navigation.m_change is None or navigation.obj_datas is not None:
```

Comment:
The SendTargetToRobot is always returning false. This condition "navigation.obj_datas is not None" should be changed.

Reply:
Yes, this was a typo. The "not None" should be "None"